Add description for getLocationSummaryList API (#66)
instana/openapi: spec/descriptions/tagSyntheticResults.md

}
```

## Get a list of Synthetic locations with Last Test Run on (each location) data
The endpoint returns a list of Synthetic locations with Last Test Run on (each location) result data

### Mandatory Parameters

**timeFrame** As in our UI you can specify the timeframe for metrics retrieval.
```
  windowSize           to
     (ms)       (unix-timestamp)
<----------------------|

"timeFrame": {
	"windowSize": 60000,
	"to": {current timestamp}
}
```

### Optional Parameters

> **pagination** if you use pagination you most probably want to fix the timeFrame for the retrieved metrics

Reviewer: Do you mean that if they use pagination, they should also specify a timeframe?  If so then may be it could say "if you use pagination, you should provide the timeframe parameter".  I am not sure what you mean by "fix the timeFrame", unless you just mean to have a set timeframe?

Reviewer 2: I think "fix the timeFrame" means you have to use the same timeFrame for all the pages you want to query. 
If you change it, the elements of the pagination will probably change too.

Reviewer: that is more clear to me

> **pagination** if you use pagination you most probably want to fix the timeFrame for the retrieved metrics

Reviewer: ```suggestion
**pagination** if you use pagination you should use the same timeFrame for all of the pages you want to query
```

1. *page* select the page number you want to retrieve
2. *pageSize* set the number of Synthetic locations you want to return with one query

> **order** You can order the returned items alphanumerical by label, either ascending or descending

Reviewer: ```suggestion
**order** You can order the returned items alphanumerically by label, either ascending or descending
```
